drivers/sensor/qdec_nrfx/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+zephyr_library()
```

Comment:
Add license/copyright header to all new files, not just .c/.h

Reply:
Fixed